cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/io/cdap/cdap/etl/api/connector/ConnectorSpec.java

```diff
@@ -78,30 +88,31 @@ public static Builder builder() {
    */
   public static class Builder {
     private Schema schema;
-    private Set<PluginSpec> relatedPlugins;
+    private final Set<PluginSpec> relatedPlugins;
+    private final Set<SampleType> supportedSampleTypes;
 
     public Builder() {
       this.relatedPlugins = new HashSet<>();
+      this.supportedSampleTypes = new HashSet<>();
     }
 
     public Builder setSchema(@Nullable Schema schema) {
       this.schema = schema;
       return this;
     }
 
-    public Builder setRelatedPlugins(Set<PluginSpec> relatedPlugins) {
```

Comment:
Any specific reason to drop this method?

Reply:
No, there wasn't, I think I just noticed that it wasn't used anywhere. Probably best not to touch existing code, so I added it back in.